app/decorators/book_decorator.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+class BookDecorator < Draper::Decorator
+  BOOK_DESCRIPTION = 240
+  delegate_all
+
+  def book_matarial
+    material.capitalize
+  end
+
+  def description_short
+    description.split('.').first
+  end
+
+  def dimensions
+    I18n.t('books.show.dimensions', height: height, width: width, depth: depth)
+  end
+
+  def author_name
+    "#{author.first_name} #{author.last_name}"
```

Comment:
I think this part of code related to the `Author` decorator
Propose to move it to `AuthorDecorator` and use `decorates_association :author` method here

```ruby
class AuthorDecorator
  def full_name
    "#{first_name} #{last_name}"
  end
end

class BookDecorator
  decorates_association :author

  delegate :full_name, to: :author, prefix: true
end

book.author_full_name
```